packages/react-tabs/Spec.md

```diff
@@ -63,54 +63,60 @@ The existing components are:
 
 ### Approach
 
-v0 provides tabs, toolbars, menus, and even breadcrumbs with the Menu components.
-v0 supports tabs behavior when accessibility={tabListBehavior}
+v0 Menu provides tabs, toolbars, menus, and breadcrumbs.
+v0 Menu supports tabs interaction when accessibility={tabListBehavior}
 
 v8 provides a separate Pivot component with tabs behavior and appearance customization.
 
-### Appearance
+### Appearances
 
 #### **Pivot**
 
 A 'Pivot' appears as a horizontal set of borderless buttons with an underline indicating current selection.
-Both v0 and v8 provide a 'Pivot' appearance.
-v8 by default and v0 when underlined and primary props are true.
+
+v0 and v8 provide a 'Pivot' appearance.
+v0: underlined=true
+v8: by default
 
 #### **Block Tab**
 
 A 'Block Tab' appears as a horizontal set of borderless buttons.
 The selection has an active color, rectangular background.
 
 Both v0 and v8 provide the 'Block Tab' appearance.
-v0 by default and v8 when the linkFormat prop equals "tabs".
+v0: by default
+v8: linkFormat="tabs"
 
 #### **Varations**
 
-v0 supports decorating tabs with arrows pointing toward the associated content when the pointing prop is set.
+v0 supports decorating tabs with arrows pointing toward the associated content.
+v0: pointing=true
 
 v8 supports a large tab style with increased padding per tab.
+v8: linkSize="large"
 
 ### Tab Item Content
 
 Both v0 and v8 support text, icon, and text and icon as tab content.
 v0 additionally supports menus as tab content.
 
-Both v0 and v8 support custom rendering of tab content through a render props callbacks per item.
+Both v0 and v8 support custom rendering of tab content through a render props per item.
 
 ### Overflow
 
-v0 support overflow when behaving as a toolbar with the accessibility={menuAsToolbarBehavior}.
-This is mutually exclusive with the accessibility={tabListBehavior}.
+v0 support overflow when behaving as a toolbar.
```

Comment:
I would propose that we do not support item overflow as a part of the component. In the past this has made v0 toolbar incredibly complicated and hard to maintain. The fact that such a app specific feature was walled off behind our API meant that all manner of hacks were born on the application side.

The overflow logic also might not be appropriate for each app's use case (visually hide elements, or stop rendering them ?). 

Here are some examples of appbars/toolbars in the open and none of them provide an automatic overflow feature, but do document easy ways to add an overflow menu to the component and let users handle the actualy overflow.

* https://mui.com/components/app-bar/#app-bar-with-menu
* https://reakit.io/docs/toolbar/#toolbar-with-menu
* https://react-bootstrap.github.io/components/navs/#using-dropdowns

Other libraries after doing a brief survey, just don't support this feature

* https://www.radix-ui.com/docs/primitives/components/toolbar
* https://www.radix-ui.com/docs/primitives/components/tabs
* https://chakra-ui.com/docs/disclosure/tabs
* https://evergreen.segment.com/components/tabs
* https://react-spectrum.adobe.com/react-spectrum/Tabs.html
* https://react-bootstrap.github.io/components/tabs/

Reply:
Just checked the design spec, it does not actually have requirements on the actual overflow itself but just the dropdown menu.. this seems like an even greater sign that we should not support this feature baked into the components

Reply:
+1 to what was mentioned above. Unless we have a mandatory requirement, not having a design spec on this and having this work for all possible cases seems like a feature that will grow forever.
I would vote for a lean approach and not have it, if possible.